INSTALL.md

```diff
@@ -1,143 +1,120 @@
-## Installation Guide
-
-### **Instruction to install PISA on your local machine**
```

Comment:
Mention that commands assume bash shell (see https://github.com/icecube/pisa/blob/f8ce002424791a273c4bdea986a501e07418e25b/INSTALL.md).